lib/Calendar.js

```diff
@@ -0,0 +1,91 @@
+"use strict";
+
+const xmlParser = require("./xml/parser");
+const ejs = require("ejs");
+const moment = require("moment");
+
+const xml = require("./xml");
+
+const byTimeTemplate = ejs.compile(xml.byTime);
+const multigetTemplate = ejs.compile(xml.multiget);
+
+function createCalendar(request) {
+    function Calendar(config) {
```

Comment:
You can write Calendar es a ES2015-`Class`. Check out this articl: http://www.2ality.com/2015/02/es6-classes-final.html